docs/Customization/Incorporating-HLS.rst

```diff
@@ -0,0 +1,69 @@
+.. _incorporating-hls:
+
+Incorporating HLS
+============================
+
+High Level Synthesis (HLS) is a method for iterating quickly on 
+different hardware algorithms that automatically generates an RTL 
+circuit to match a specification in a high level language like C.
+
+Here, we will integrate an HLS-generated accelerator that computes 
+the Great Common Denominator (GCD) of two integers. This tutorial 
+builds on the sections :ref:`mmio-accelerators` and 
+:ref:`incorporating-verilog-blocks`. The code for this example can 
+be found in ``/generators/hls-example``
+
+Adding an HLS project
+---------------------------------------
+
+In this tutorial, we use Vitis HLS, version 2023.2. 
```

Comment:
The CI is using an older version of the tool FYI.

Reply:
Also maybe I would point to some docs/link to Vitis HLS. Isn't there restrictions on where this can be used (and if so, should we mention something about that here)?

Reply:
I looked closer on the product site and the documentation but I couldn't find any restrictions on where Vitis HLS can be used. Possibly it's in the license somewhere, but I don't really want to crawl through that... Some forum comments implied that Vitis HLS has been used in commercial tapeouts, so if there are rules they don't seem to be very strict. We should be able to just leave it up to the user to know what they're doing.